app/views/candidate_interface/apply/show.html.erb

```diff
@@ -0,0 +1,47 @@
+<% content_for :title, t('page_titles.application') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      <%= t('apply.heading') %>
+    </h1>
+    <p class="govuk-body">
+      You must apply for this course on UCAS. You’ll need to register with UCAS
+      before you can apply. Visit Get into Teaching for
+      <a class="govuk-link" target="_blank" href="https://getintoteaching.education.gov.uk">
+        guidance on applying for teacher training</a>.
+    </p>
+
+    <p class="govuk-body">
+      When you'll apply you'll need these codes for the Choices section of your
```

Comment:
```suggestion
      When You’ll apply you'll need these codes for the Choices section of your
```

(For consistency with above)

Reply:
@alexdesi this is the design, yeah there isn't emphasis at the moment: https://bat-design-history.herokuapp.com/public/images/find-teacher-training/choose-how-to-apply-2/apply-for-this-course-(without-choice).png